f5-sdk-dist/scripts/configure.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# Copyright 2017 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import errno
+import glob
+import json
+import os
+import re
+import sys
+
+
+def add_package(pkg_type, pkg_name, working=None):
+    """add_package
+
+Adds a package to the existing config.JSON file.  This is a standalone function
+to handle this functionality.  The existing config.JSON is read in and is left
+unchanged.
+    """
+    kvp = {pkg_type: pkg_name}
+    working = os.getcwd() if not working else os.path.abspath(working)
+    if '-dist/script' in working:
+        config = working + "/config.JSON"
+    elif '-dist' in working:
+        config = working + "/script/config.JSON"
+    else:
+        config = working + "/*-dist/script/config.JSON"
+        entropy = glob.glob(config)
```

Comment:
The use of `entropy` is thought provoking and I like that.   I think the cost to readability should be considered.   Should every reader of this code be required to understand the concept of entropy?   What's the purpose of the variable name?